db/seeds.rb

```diff
@@ -7,49 +7,73 @@ def create_user!(name)
   )
 end
 
+def create_developer!(attributes, avatar_name:)
```

Comment:
If `avatar_name:` is required we can pass it with `attributes` so we don't have to use two hashes when calling this method.

```
def create_developer!(attributes)
  avatar_name = attributes.delete(:avatar_name)
  Developer.create!(attributes) do |developer|
    developer.avatar.attach(io: File.open(Rails.root.join("test/files/#{avatar_name}")), filename: avatar_name)
  end
end
```